### ISSUE TITLE
[BUG] `sequence item 1: expected str instance, Attribute found`

### ISSUE THREAD

**Issue by AlecHaring** [dashingsoft/pyarmor#1051]
**Executed command**: `pyarmor gen --enable-rft .\main.py`

Output:
```
INFO     Python 3.9.12
INFO     Pyarmor 8.0.22 (pro), ...
INFO     search inputs ...
INFO     find script main.py
INFO     find 1 top resources
INFO     start to generate runtime files
INFO     start to obfuscate scripts
INFO     process resource "main"
INFO     obfuscating file main.py
CRITICAL unknown error, please check pyarmor.error.log
ERROR    sequence item 1: expected str instance, Attribute found
```

**pyarmor.error.log:**
```
3580 MainProcess 2023-03-30 11:48:11,988
Traceback (most recent call last):
  File "C:\Users\alec\Desktop\PythonProjects\pyarmor-test\venv\lib\site-packages\pyarmor\cli\__main__.py", line 585, in main
    main_entry(sys.argv[1:])
  File "C:\Users\alec\Desktop\PythonProjects\pyarmor-test\venv\lib\site-packages\pyarmor\cli\__main__.py", line 573, in main_entry
    args.func(ctx, args)
  File "C:\Users\alec\Desktop\PythonProjects\pyarmor-test\venv\lib\site-packages\pyarmor\cli\__main__.py", line 152, in cmd_gen
    builder.process(options, pack=args.pack)
  File "C:\Users\alec\Desktop\PythonProjects\pyarmor-test\venv\lib\site-packages\pyarmor\cli\generate.py", line 153, in process
    self._obfuscate_scripts()
  File "C:\Users\alec\Desktop\PythonProjects\pyarmor-test\venv\lib\site-packages\pyarmor\cli\generate.py", line 118, in _obfuscate_scripts
    code = Pytransform3.generate_obfuscated_script(self.ctx, r)
  File "C:\Users\alec\Desktop\PythonProjects\pyarmor-test\venv\lib\site-packages\pyarmor\cli\core\__init__.py", line 48, in generate_obfuscated_script
    return m.generate_obfuscated_script(ctx, res)
  File "<maker>", line 714, in generate_obfuscated_script
  File "<maker>", line 504, in process
  File "<maker>", line 504, in <listcomp>
  File "<rftmaker>", line 17, in process
  File "<rftmaker>", line 329, in process
  File "<rftmaker>", line 305, in _refactor_tree
  File "<rftmaker>", line 297, in visit
  File "<rftmaker>", line 297, in visit
  File "<rftmaker>", line 297, in visit
  [Previous line repeated 3 more times]
  File "<rftmaker>", line 289, in visit
  File "<rftmaker>", line 249, in _rename_attrnode
TypeError: sequence item 1: expected str instance, Attribute found
```

**main.py**
```python
import traceback
import inspect


def function_one():
    pass


def function_two():
    pass


global_variable_one = 1
global_variable_two = 2


def dummy_exception():
    try:
        inner_exception()
    except:
        traceback.print_exc()


def inner_exception():
    raise Exception("Test exception")


def testing():
    dummy_exception()

    print("Globals:")
    for name, obj in globals().items():
        if name.startswith('__'):
            continue
        global_type = type(obj).__name__
        print(f"\t{global_type} - {name}")

    print(f"Current function name: {inspect.currentframe().f_code.co_name}")


if __name__ == "__main__":
    testing()
```

It seems like the error is related to `inspect.currentframe().f_code.co_name`, more specifically the `f_code.co_name` part. Removing it fixes the problem

I understand this error was reported in #1048, but I thought I'd create a new issue dedicated to it, since #1048 talks about multiple issues.

**Comment by jondy** [comment]
Got it, thanks. I'll check it.

**Comment by AlecHaring** [comment]
Seems to have been fixed in version 8.0.26 :)
